src/assets/main.scss

```diff
@@ -40,18 +45,21 @@ $active-node-color: #0076d6;
         width: 100%;
         display: flex;
         overflow:hidden;
-        > * {
+        padding: .5rem .75rem;
+        > .title {
             flex-grow: 1;
-            display: block;
-            padding: .5rem .75rem;
+            display: -webkit-box;
             overflow:hidden;
             text-overflow: ellipsis;
-            white-space: nowrap;
+            overflow-wrap: break-word;
+            -webkit-line-clamp: 2;
+            -webkit-box-orient: vertical;
```

Comment:
Any reason for using a Chrome-specific extension? And does this behave as expected on non-Chrome browsers?